lib/templates/lab5/main.dart

```diff
@@ -1,13 +1,165 @@
-// Use these dependencies for your classes
 import 'dart:convert';
+import 'package:education/livecoding/json/run.dart';
+import 'package:education/livecoding/navigation/main.dart';
+import 'package:flutter/material.dart';
+import 'package:flutter/widgets.dart';
 import 'package:http/http.dart';
 import 'package:json_annotation/json_annotation.dart';
 import 'package:freezed_annotation/freezed_annotation.dart';
+import 'package:flutter_web_plugins/flutter_web_plugins.dart';
 
+import 'post.dart';
+import 'comment.dart';
+import 'user.dart';
 
-void main() {}
+void configureApp() {
+  setUrlStrategy(PathUrlStrategy());
+}
 
-// TODO add needed classes for Flutter APP
+void main() {
+  configureApp();
+  runApp(const MainPage());
+}
+
+final Map<String, String> routes = {
+  '/': '/one',
+  '/second': '/second',
+  '/third': '/third',
+  '/generated': '/generated',
+};
+
+class MainPage extends StatelessWidget {
+  const MainPage({super.key});
+
+  @override
+  Widget build(BuildContext context) {
+    return MaterialApp(
+      title: 'Lab 5',
+      theme: ThemeData(
+        primarySwatch: Colors.blue,
+      ),
+      home: HomePage(),
+      onGenerateRoute: (settings) {
+        if (settings.name == '/generated') {
```

Comment:
should be possible to put statusCode not as an argument - as an number in source path